### PR TITLE
versioning and publishing workflow

### PR DESCRIPTION
**Pull request by dianelee217**
## What
  * Added CI/CD workflow for publishing package to pypi
  * Updated versioning
  * Added project classifiers

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,9 +25,9 @@
 
 setup(
     name="rubicon",
-    version=get_version(),
-    author="The Rubicon Team",
-    author_email="rubicon-developers@capitalone.com",
+    version=__version__,
+    author="Joe Wolfe, Ryan Soley, Diane Lee, CapitalOne",
+    license='Apache License, Version 2.0',
```
Comment (mmccarty) [inline setup.py:30]: Please add me here. I do intend to contribute.

**Comment by mmccarty** [comment]
Can you also add badges with CI status to the README?

File: rubicon/version.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""
+File containers the version number for the package
+"""
+
+MAJOR               = 0
+MINOR               = 1
+MICRO               = 0
+
+VERSION             = '%d.%d.%d' % (MAJOR, MINOR, MICRO)
```
Comment (joe-wolfe21) [inline rubicon/version.py:9]: @mmccarty - we suggested starting at `v0.1.0`  - do you agree ?

File: setup.py [hunk 3]
```diff
@@ -41,4 +41,27 @@
             "rubicon_experiment = rubicon.intake_rubicon.experiment:ExperimentSource",
         ],
     },
+    classifiers=[
+        # How mature is this project? Common values are
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 4 - Beta',
```
Comment (joe-wolfe21) [inline setup.py:49]: @mmccarty - we were debating between Beta, or Production/Stable -- what do you think?

Comment (mmccarty) [reply setup.py:47]: I think Beta is fine for now given that we are just now open sourcing and might get some activity that would make things less stable.

**Review by joe-wolfe21 — APPROVED** [review]
LGTM

**Pull request merged**